### ISSUE TITLE
Document dataset "purpose"

### ISSUE THREAD

**Issue by breznak** [breznak/ML.benchmark#9]
For new people reviewing this repo, understanding and navigating in the specific dataset sub-sets might be confusing, such as 
https://github.com/breznak/ML.benchmark/tree/master/data/datasets/corruptedData/sectionAnomaly/amplitude
Therefore I'd like to suggest the following: 
- for each path, include a small (few lines) Readme what type of data it contains, eg: 

```
corruptedData: data with some kind of (synthetic) anomaly added. 
sectionAnomaly: the anomaly occurs on a continuous subsection of the data stream (eg on time interval T=[9:00,11:30]). Other types of anomaly are [pointAnomaly](link) and "system generator anomaly"
aplitude: Anomaly caused by changed (increased, decreased) amplitude of the function. See image. 
```
- for the "leaf nodes" of the anomaly path (corruptedData/sectionAnomaly/amplitude)  include an image to get easier grasp of the anomaly type at hand. 
- additionally, these "anomaly types" could also be described in the high level "datasets/ Readme"

CC @wattik ?

**Comment by breznak** [comment]
@wattik is this fixed in PR #10 ?

**Comment by breznak** [comment]
Fixed better in #17
